meson.build

```diff
@@ -36,6 +36,7 @@ prefixdir = get_option('prefix')
 
 conf.set_quoted('CONFIG_H_SYSCONFDIR', join_paths(prefixdir, get_option('sysconfdir')))
 conf.set_quoted('CONFIG_H_DATADIR', join_paths(prefixdir, get_option('datadir')))
+conf.set_quoted('HIRTE_VERSION', meson.project_version())
```

Comment:
I'd still prefer to align with other names in `config.h`:

```
conf.set_quoted('CONFIG_H_PROJECT_VERSION', meson.project_version())
```

Reply:
Yes, keeping the naming consistent makes sense. And by using the `CONFIG_H_` prefix, we have a "hint" in the code where this constant is coming from - since we dynamically link without having an explicit `#include`.